src/test/java/SetTest.java

```diff
@@ -0,0 +1,53 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.converter.ConvertWith;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 크기를 확인한다.")
+    void countSizeOfSet(){
+        // WHEN
+        int size = numbers.size();
+
+        // THEN
+        int expectedSize = 3;
+        assertThat(size).isEqualTo(expectedSize);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    @DisplayName("예상되는 값이 존재하는지 확인한다.")
+    void checkIfHasNums(int input){
+        // WHEN & THEN
+        assertThat(numbers.contains(input)).isTrue();
```

Comment:
`LearningTest` 를 보니 이미 잘 알고 계신 것 같지만, assertj 에서 제공하는 contains 로 변경해도 좋을 것 같네요!
```suggestion
        assertThat(numbers).contains(input);
```